diff -ruN netcdf-fortran-4.5.4/nf03_test4/CMakeLists.txt netcdf-fortran-4.5.4-patch/nf03_test4/CMakeLists.txt
--- netcdf-fortran-4.5.4/nf03_test4/CMakeLists.txt	2022-01-07 14:43:57.000000000 -0700
+++ netcdf-fortran-4.5.4-patch/nf03_test4/CMakeLists.txt	2022-01-19 16:56:26.078419044 -0700
@@ -48,7 +48,7 @@
     build_bin_test(f90tst_parallel3 ".F90")
     build_bin_test(f90tst_nc4_par ".F90")
     build_bin_test(f90tst_parallel_fill ".f90")
-    build_bin_test(f90tst_parallel_compressed ".f90")
+    build_bin_test(f90tst_parallel_compressed ".F90")
 
     add_sh_test(F90_tests run_f90_par_test)
   ENDIF (TEST_PARALLEL)
