diff -ruN netcdf-fortran-4.6.2/CMakeLists.txt netcdf-fortran-4.6.2-patch/CMakeLists.txt
--- netcdf-fortran-4.6.2/CMakeLists.txt	2025-03-26 23:01:02.000000000 +0100
+++ netcdf-fortran-4.6.2-patch/CMakeLists.txt	2025-05-23 12:49:34.454996679 +0200
@@ -849,6 +849,8 @@
 
 IF(USE_NETCDF_V2)
   SET(BUILD_V2 ON CACHE BOOL "")
+ELSE()
+  SET(BUILD_V2 OFF CACHE BOOL "")
 ENDIF()
 
 OPTION(ENABLE_FORTRAN_TYPE_CHECKS
@@ -862,7 +864,6 @@
 
 # Set the default fortran builds; default is to build f03
 SET(BUILD_F90 "ON")
-SET(BUILD_V2 "ON")
 SET(BUILD_F03 "ON")
 
 IF(BUILD_F90)
