diff -ruN -x __pycache__ -x obj -x '*~' -x tags -x TAGS netcdf-fortran-4.6.0.orig/CMakeLists.txt netcdf-fortran-4.6.0/CMakeLists.txt
--- netcdf-fortran-4.6.0.orig/CMakeLists.txt	2022-07-29 21:37:55.000000000 +0200
+++ netcdf-fortran-4.6.0/CMakeLists.txt	2023-01-23 16:34:30.142461143 +0100
@@ -838,7 +838,7 @@
 endif()
 
 IF(USE_NETCDF_V2)
-  SET(BUILD_V2 ON CACHE BOOL "")
+  SET(BUILD_V2 OFF CACHE BOOL "")
 ENDIF()
 
 OPTION(ENABLE_FORTRAN_TYPE_CHECKS
@@ -852,7 +852,7 @@
 
 # Set the default fortran builds; default is to build f03
 SET(BUILD_F90 "ON")
-SET(BUILD_V2 "ON")
+SET(BUILD_V2 "OFF")
 SET(BUILD_F03 "ON")
 
 IF(BUILD_F90)
