diff --git a/nf03_test/f03tst_open_mem.F b/nf03_test/f03tst_open_mem.F
index d42c43d..a8c3892 100755
--- a/nf03_test/f03tst_open_mem.F
+++ b/nf03_test/f03tst_open_mem.F
@@ -122,7 +122,7 @@ C then call nf_open_mem in NF_DISKLESS mode
         Read(7, iostat=iostat) memfile(1:ncsize)
         If (iostat == 0) then
            Close(7)
-           mode = IOR(NF_DISKLESS, NF_INMEMORY)
+           mode = NF_INMEMORY
            retval = nf_open_mem(FILE_NAME, mode, ncsize, memfile, ncid)
         Else
            retval = iostat
