--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -574,12 +574,6 @@ ENDIF()
 
 IF (UNIX AND ${CMAKE_SIZEOF_VOID_P} MATCHES "8")
   ADD_DEFINITIONS(-DLONGLONG_IS_LONG)
-  # for easier debugging of cfortran.h
-  IF (APPLE)
-    IF(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-      SET (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fmacro-backtrace-limit=0")
-    ENDIF()
-  ENDIF(APPLE)
 ENDIF()
 
 # Determine C library setting for NC_RELAX_COORD_BOUND
@@ -719,7 +713,7 @@ ENDIF(ENABLE_FORTRAN_TYPE_CHECKS)
 # FILE(RENAME "${NC4F_BINARY_DIR}/nf-config.cmake.in" "${NC4F_BINARY_DIR}/nf-config")
 
 # make sure previous two files get cleaned up...
-SET_DIRECTORY_PROPERTIES (DIRECTORY PROPERTY ADDITIONAL_MAKE_CLEAN_FILES ${netcdf-fortran_BINARY_DIR}/config.h ${netcdf-fortran_BINARY_DIR}/nf-config)
+# SET_DIRECTORY_PROPERTIES (DIRECTORY PROPERTY ADDITIONAL_MAKE_CLEAN_FILES ${netcdf-fortran_BINARY_DIR}/config.h ${netcdf-fortran_BINARY_DIR}/nf-config)
 
 ##
 # Configuration for post-install RPath
@@ -842,9 +836,9 @@ IF(CMAKE_LINK_FLAGS)
 ENDIF()
 
 
-INSTALL(PROGRAMS ${NC4F_BINARY_DIR}/nf-config
-        DESTINATION "${CMAKE_INSTALL_BINDIR}"
-        COMPONENT utilities)
+# INSTALL(PROGRAMS ${NC4F_BINARY_DIR}/nf-config
+#         DESTINATION "${CMAKE_INSTALL_BINDIR}"
+#         COMPONENT utilities)
 
 INCLUDE_DIRECTORIES(${NC4F_BINARY_DIR})
 # End autotools-style checks for config.h
