--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -574,12 +574,6 @@ ENDIF()
 
 IF (UNIX AND ${CMAKE_SIZEOF_VOID_P} MATCHES "8")
   ADD_DEFINITIONS(-DLONGLONG_IS_LONG)
-  # for easier debugging of cfortran.h
-  IF (APPLE)
-    IF(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-      SET (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fmacro-backtrace-limit=0")
-    ENDIF()
-  ENDIF(APPLE)
 ENDIF()
 
 # Determine C library setting for NC_RELAX_COORD_BOUND
