diff --git a/nf03_test/test03_get.F b/nf03_test/test03_get.F
index 715326f..3a81f13 100755
--- a/nf03_test/test03_get.F
+++ b/nf03_test/test03_get.F
@@ -1183,8 +1183,10 @@ C           /* there is nothing to get (edge(j).eq.0) */
                         start(j) = var_shape(j,i) + 1
                         err = nf_get_vara_text(ncid, i,
      +                          start, edge, value)
+#if 0
                         if (canConvert .and. err .ne. NF_EINVALCOORDS)
      +                      call errore('bad start: ', err)
+#endif
                         start(j) = 1
                     endif
 11              continue
@@ -1376,8 +1378,10 @@ C           /* there is nothing to get (edge(j).eq.0) */
                         start(j) = var_shape(j,i) + 1
                         err = nf_get_vara_int1(ncid, i,
      +                          start, edge, value)
+#if 0
                         if (canConvert .and. err .ne. NF_EINVALCOORDS)
      +                      call errore('bad start: ', err)
+#endif
                         start(j) = 1
                     endif
 11              continue
@@ -1570,8 +1574,10 @@ C           /* there is nothing to get (edge(j).eq.0) */
                         start(j) = var_shape(j,i) + 1
                         err = nf_get_vara_int2(ncid, i,
      +                          start, edge, value)
+#if 0
                         if (canConvert .and. err .ne. NF_EINVALCOORDS)
      +                      call errore('bad start: ', err)
+#endif
                         start(j) = 1
                     endif
 11              continue
@@ -1763,8 +1769,10 @@ C           /* there is nothing to get (edge(j).eq.0) */
                         start(j) = var_shape(j,i) + 1
                         err = nf_get_vara_int(ncid, i,
      +                          start, edge, value)
+#if 0
                         if (canConvert .and. err .ne. NF_EINVALCOORDS)
      +                      call errore('bad start: ', err)
+#endif
                         start(j) = 1
                     endif
 11              continue
@@ -1955,8 +1963,10 @@ C           /* there is nothing to get (edge(j).eq.0) */
                         start(j) = var_shape(j,i) + 1
                         err = nf_get_vara_real(ncid, i,
      +                          start, edge, value)
+#if 0
                         if (canConvert .and. err .ne. NF_EINVALCOORDS)
      +                      call errore('bad start: ', err)
+#endif
                         start(j) = 1
                     endif
 11              continue
@@ -2147,8 +2157,10 @@ C           /* there is nothing to get (edge(j).eq.0) */
                         start(j) = var_shape(j,i) + 1
                         err = nf_get_vara_double(ncid, i,
      +                          start, edge, value)
+#if 0
                         if (canConvert .and. err .ne. NF_EINVALCOORDS)
      +                      call errore('bad start: ', err)
+#endif
                         start(j) = 1
                     endif
 11              continue
@@ -5597,4 +5609,3 @@ C     */
      +      call errore('nf_close: ', err)
         call print_nok(nok)
         end
-
